Makefile

```diff
@@ -45,4 +45,6 @@ update-test-data:
 	./run_duden.py --export Kragen > tests/test_data/Kragen.yaml
 	./run_duden.py --export Petersilie > tests/test_data/Petersilie.yaml
 	./run_duden.py --export einfach -r1 > tests/test_data/einfach.yaml
-	./run_duden.py --export Keyboard > tests/test_data/Keyboard.yaml
\ No newline at end of file
+	./run_duden.py --export Keyboard > tests/test_data/Keyboard.yaml
+	./run_duden.py --export Meme > tests/test_data/Meme.yaml
+	
```

Comment:
This is a triviality, there's an extra tab at the end of the file. Ideally, the file would end with a single newline.